### DISCUSSION TITLE
Why is "frameRateMatching" defaultValue="false" ignored?

### DISCUSSION THREAD

**Discussion by brentonv** [moneytoo/Player#212] [General]
When building the app it does not apply the "frameRateMatching" defaultValue="false" in root_preferences.xml

Can you please explain why this happens?

Thanks

**Comment by moneytoo** [comment]
There are additional runtime conditions that change that default:

https://github.com/moneytoo/Player/blob/4330e6ef0ed2ad77698961c5f06deeca31480598/app/src/main/java/com/brouken/player/Prefs.java#L73

**Reply by brentonv** [reply]
Thank you @moneytoo  I can see how it's working🙂
